default/python/AI/AIstate.py

```diff
@@ -446,9 +446,6 @@ def __update_system_status(self):
             self.systemStatus[sys_id]["myFleetRating"] = 0
             self.systemStatus[sys_id]["myFleetRatingVsPlanets"] = 0
 
-        # for use in debugging
-        verbose = False
-
```

Comment:
The idea behind this obviously was that a developer can easily switch on some further debug messages.
I haven't started digging into threat calculations, but I guess sooner or later I will and I might miss those messages.

Perhaps we should generally make debugging configurable, as I started here:
https://github.com/freeorion/freeorion/blob/882933511cbbeb438c49faf7db7f3a828812e217/default/python/AI/colonization/colony_score.py#L12

Reply:
> Perhaps we should generally make debugging configurable, as I started here:

That is a good idea.  So you can implement it and add meaningfully debug messages when you'll start digging.

Reply:
The fact that I haven't started using them doesn't mean they are useless, please do not remove them.
If you want to get rid of the warning, remove the boolean, or set via configuration.